testing/kea-hook-runscript/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Maintainer: Steve HOLWEG <skytep@gmail.com>
+
+pkgname=kea-hook-runscript
+pkgver=1.1.0
+pkgrel=0
+pkgdesc="This a hook for the Kea DHCP server that allows to run an external script."
+arch="all"
+url="https://github.com/zorun/kea-hook-runscript"
+license="MPL2"
+depends="kea"
+makedepends="kea-dev kea-utils boost-dev"
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/zorun/$pkgname/archive/v$pkgver.tar.gz"
+options="!check" # Library and makefile provide no tests
+
+build() {
+	cd "$builddir"
+
+	make
+}
+
+package() {
+	cd "$builddir"
+
+	mkdir -p ${pkgdir}/usr/lib/hooks/
+
+	mv kea-hook-runscript.so ${pkgdir}/usr/lib/hooks/kea-hook-runscript.so
```

Comment:
Please don't move files from builddir (it makes hard to rerun package phase), copy them.